clients/go/pkg/pb/gateway.pb.go

```diff
@@ -121,7 +121,7 @@ func (x Partition_PartitionBrokerHealth) Number() protoreflect.EnumNumber {
 
 // Deprecated: Use Partition_PartitionBrokerHealth.Descriptor instead.
 func (Partition_PartitionBrokerHealth) EnumDescriptor() ([]byte, []int) {
-	return file_gateway_proto_rawDescGZIP(), []int{33, 1}
+	return file_gateway_proto_rawDescGZIP(), []int{39, 1}
```

Comment:
❓ What does this number do?

Reply:
And all the other random number changes 😄

Reply:
Is this partly auto-generated or something 🤔

Reply:
yes, it was auto-generated. I have no idea why it was changed.